### DISCUSSION TITLE
Customize title for header's custom homepage URL used with the header icon

### DISCUSSION THREAD

**Discussion by mikkvitchko** [squidfunk/mkdocs-material#2578] [Ideas]
Currently the site_name is used as "link title" (`header.html`), and since its not customizable sometimes this leads to an issue.

```
...
<a href="{{ config.extra.homepage | d(nav.homepage.url, true) | url }}" title="{{ config.site_name | e }}" 
...
```
 For those who change the URL using `config.extra.homepage` it is also expected to change the title, as URL leads to some other site, not to the site with documentation. Therefore the use of `config.site_name` for the title in this case may not be appropriate.

I understand that this can be solved by using `overrides\partials\header.html` - but  I think that some less dramatic customization option might be helpful :)

**Comment by squidfunk** [comment]
We can't provide flags or options for everything. I understand that this might cause you some trouble overriding, but I don't want to make the template more complicated than it currently already is. Overrides are simple and breaks in templates are limited to major versions. Most changes these days are related to CSS or JS. Making stuff more complicated means more work for me answering issues, which steals my time from working on bugfixes and features 😅

After all, were talk of about parametrizing the value of the tooltip which appears when hovering over the logo.
